ci_framework/playbooks/06-deploy-edpm.yml

```diff
@@ -7,28 +7,38 @@
 - hosts: "{{ cifmw_target_host | default('localhost') }}"
   gather_facts: false
   tasks:
-    - name: Call predeploy
+    - name: Perform Podified and EDPM deployment on compute nodes provisioned with bmaas
+      when: edpm_compute_baremetal | default('false') | bool
+      ansible.builtin.include_role:
+        name: edpm_deploy_baremetal
+
+    - name: Deploy podified control plane
+      when: not edpm_compute_baremetal | default('false') | bool
       ansible.builtin.include_role:
         name: edpm_prepare
 
 - name: Run post_ctlplane_deploy hooks
+  when: not edpm_compute_baremetal | default('false') | bool
   vars:
     hooks: "{{ post_ctlplane_deploy | default([]) }}"
     step: post_ctlplane_deploy
   ansible.builtin.import_playbook: ./hooks.yml
 
-- hosts: "{{ cifmw_target_host | default('localhost') }}"
+- name: EDPM deployment on pre-provisioned VMs
+  hosts: "{{ cifmw_target_host | default('localhost') }}"
   gather_facts: false
   tasks:
-    - name: Create and provision external computes
-      ansible.builtin.include_role:
-        name: libvirt_manager
-        tasks_from: deploy_edpm_compute.yml
+    - block:
+      - name: Create and provision external computes
+        ansible.builtin.include_role:
+          name: libvirt_manager
+          tasks_from: deploy_edpm_compute.yml
 
-    - name: Deploy EDPM
-      ansible.builtin.include_role:
-        name: edpm_deploy
-      when: deploy_edpm | default('false') | bool
```

Comment:
same - `cifmw_` prefix needed - and would be good to move the `when` right after the `name` for an easier reading

Reply:
err, that was a removal... sorry.